internal/ethapi/api.go

```diff
@@ -2385,7 +2385,7 @@ func (s *BundleAPI) CallBundle(ctx context.Context, args CallBundleArgs) (map[st
 		}
 		uint256GasPrice, ok := uint256.FromBig(gasPrice)
 		if !ok {
-			return nil, fmt.Errorf("err: %w; txhash %s", err, tx.Hash())
```

Comment:
I feel like someone can depend on this particular message struct. 

Is it possible to not change it?

Reply:
what if:

```go
return nil, fmt.Errorf("err: incorrect gasPrice; txhash %s", tx.Hash())
```

Reply:
ok, as I undestood this is the code path that was added for deneb so no need to make this compatible